nixos-module.nix

```diff
@@ -30,6 +30,7 @@ in
 
         discovery.bootstrap = lib.mkOption {
             type = lib.types.listOf lib.types.str;
+            default = [ "testnet-bootstrap.alephium.org:9973" ];
             description = ''
                 Addresses of other Alephium nodes to connect to initially.
                 This is different for each network, so there is no default.
```

Comment:
This description is now outdated. It should explain what the default value means.

Reply:
Actually the default value of this option should depend on which network-id is given, right? I don’t know if NixOS option defaults can be conditional.

Reply:
Yep indeed, I tried to hack a bit without success.
It can be conditional but the problem it's that when declaring `nixos-module.nix` we don't know yet what will be the `network-id`.
Closing that PR.